.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Remove this file @BadduCoder @ethicalrushi

Reply:
Deleted, it is an outdated file